packit/jobs.py

```diff
@@ -388,10 +388,14 @@ def handle_release(self):
         )
 
         # report
+        commit_sha = self.project.get_sha_from_tag(tag_name)
         msg = f"Copr build(ID {build_id}) triggered\nMore info: {repo_url}"
-        self.project.commit_comment(
-            commit=self.project.get_sha_from_tag(tag_name), body=msg
+        self.project.commit_comment(commit=commit_sha, body=msg)
+        build_result = api.watch_copr_build(
+            build_id, self.job.metadata.get("targets") or 60 * 60 * 2
         )
+        msg = f"Copr build {build_id} ended with result {build_result}"
```

Comment:
in a follow-up PR, we should provide instructions how people can install the packages

Reply:
I am now getting rid of the commit/PR comments in follow up and replacing them with status - would you recommend to add comment anyway with the instructions after (first?) successfull build?